features/nvstore/TESTS/nvstore/functionality/main.cpp

```diff
@@ -378,6 +377,27 @@ static void nvstore_basic_functionality_test()
     delete[] nvstore_testing_buf_get;
 }
 
+static void calc_thread_stack_size(int &num_threads, uint32_t min_size, uint32_t max_size,
```

Comment:
Please add a comment to the function about the parameters. There two in/out parameters and the idea of the function is good but not trivial. Beside that looks OK to me

Reply:
+1, I was reading the definition how the test gets num threads and stack size (I knew it would be via &), plus what heap_get returns if heap stats are not enabled (memset to 0, thus the check in reserved_size) - these were not obvious to me , neither then what while does in the below